readthedocs/projects/tasks.py

```diff
@@ -184,6 +185,7 @@ def run(self, pk, version_pk=None, build_pk=None, record=True, docker=False,
 
         if self.build_env.failed:
             self.send_notifications()
+        build_complete.send(sender=Build, build=self.build_env.build)
```

Comment:
Any reason not to send the whole environment. Seems like it would be more useful in general.